### PR TITLE
[sc-memory][python] Rename action keynodes #354

### LINKED ISSUE #354: Implement corrent action classes within ScAgentAction
Change action classes in accordance with document (see identifiers table):
https://docs.google.com/document/d/1uAeOyXOKsT4DGjuCmG8JuTakz13LyXvXuBwsg_blA8E/edit

For now use current identifiers from IMS with "question_" prefix.

### PR DESCRIPTION
**Pull request by deniskoronchik**
* [ ] Read PR [documentation](http://ostis-dev.github.io/sc-machine/dev/pr/)
* [x] Update changelog
* [ ] Update documentation

Resolves #354

### CODE REVIEW

File: sc-memory/sc-memory/sc_keynodes.hpp [hunk 1]
```diff
@@ -31,17 +31,17 @@ class ScKeynodes : public ScObject
...
   static _SC_EXTERN size_t GetRrelIndexNum();
 
-  SC_PROPERTY(Keynode("command_state"), ForceCreate(ScType::NodeConstClass))
-  _SC_EXTERN static ScAddr kCommandStateAddr;
+  SC_PROPERTY(Keynode("actions_state"), ForceCreate(ScType::NodeConstClass))
+  _SC_EXTERN static ScAddr kActionState;
 
-  SC_PROPERTY(Keynode("command_initiated"), ForceCreate(ScType::NodeConstClass))
-  _SC_EXTERN static ScAddr kCommandInitiatedAddr;
+  SC_PROPERTY(Keynode("action_initiated"), ForceCreate(ScType::NodeConstClass))
```
Comment (ShunkevichDV) [inline sc-memory/sc-memory/sc_keynodes.hpp:37]: In #357 I've asked to use modern state names, i.e. question_inititaed, etc.
action_... won't integrate with KB and all other agents right now, renaming question_ to action_ is another separate task.
For now it's better to use Action word in C++ code, but use quiestion_.. as system identifiers

**Review by ShunkevichDV — CHANGES_REQUESTED** [review]

File: sc-memory/sc-memory/kpm/sc_agent.cpp [hunk 2]
```diff
@@ -61,7 +61,7 @@ sc_result ScAgentAction::Run(ScAddr const & listenAddr, ScAddr const & edgeAddr,
     if (m_memoryCtx.HelperCheckEdge(m_cmdClassAddr, cmdAddr, ScType::EdgeAccessConstPosPerm))
```
Comment (ShunkevichDV) [inline sc-memory/sc-memory/kpm/sc_agent.cpp:61]: cmdAddr -> actAddr or actionAddr?

File: sc-memory/sc-memory/sc_keynodes.hpp [hunk 3]
```diff
@@ -31,17 +31,17 @@ class ScKeynodes : public ScObject
...
 
-  SC_PROPERTY(Keynode("command_in_progress"), ForceCreate(ScType::NodeConstClass))
-  _SC_EXTERN static ScAddr kCommandProgressdAddr;
+  SC_PROPERTY(Keynode("action_running"), ForceCreate(ScType::NodeConstClass))
+  _SC_EXTERN static ScAddr kActionRunning;
 
-  SC_PROPERTY(Keynode("command_finished"), ForceCreate(ScType::NodeConstClass))
-  _SC_EXTERN static ScAddr kCommandFinishedAddr;
+  SC_PROPERTY(Keynode("action_finished"), ForceCreate(ScType::NodeConstClass))
+  _SC_EXTERN static ScAddr kActionFinished;
```
Comment (ShunkevichDV) [inline sc-memory/sc-memory/sc_keynodes.hpp:45]: Need to discuss about finished_successfully\unsuccessfully

**Review by ShunkevichDV — CHANGES_REQUESTED** [review]

File: sc-memory/sc-memory/sc_keynodes.hpp (hunk 1 above)
Comment (deniskoronchik) [reply sc-memory/sc-memory/sc_keynodes.hpp:37]: We can replace `question_` keynodes in KB with find/replace

**Pull request closed without merging**